### PR TITLE
Merge giddh2 - beta

### PR DESCRIPTION
**Pull request by ravinderwalkover**
Merge giddh2 - beta

### CODE REVIEW

File: apps/web-giddh/src/app/audit-logs/audit-logs.component.scss [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
     justify-content: center;
-}
\ No newline at end of file
+}
+
+.clear-filter{
+    color: $midnight-blue;
+    cursor: pointer;
+}
+// @media(min-width:1366px){
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/audit-logs/audit-logs.component.scss:12]: remove commented CSS code if not required

File: apps/web-giddh/src/app/audit-logs/audit-logs.component.scss [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+//     .audit-log-sidebar{
+//          max-width: 350px;
+//     }
+// }
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/audit-logs/audit-logs.component.scss:16]: remove commented CSS code if not required

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-form/audit-logs-form.component.html [hunk 3]
```diff
@@ -1,43 +1,45 @@
...
+        <label>User</label>
+        <sh-select [options]="auditLogFormVM.users$ | async" [customFilter]="customUserFilter" [isFilterEnabled]="true" [placeholder]="'Select Option'" [(ngModel)]="auditLogFormVM.selectedUserUniqueName" [forceClearReactive]="forceClearUser$ | async"></sh-select>
+    </div>
+    <div class="d-flex">
+        <div class="form-btn-field">
+            <label class="d-block">&nbsp;</label>
+            <button class='btn btn-success mr-0' [ladda]="auditLogFormVM.getLogsInprocess$ | async" [disabled]="!auditLogFormVM.selectedEntity" (click)='getLogFilters()'>Get Logs
+            </button>
+        </div>
+        <!--<div class="form-btn-field">
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/audit-logs/components/audit-logs-form/audit-logs-form.component.html:37]: remove commented code if not required

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.ts [hunk 4]
```diff
@@ -46,7 +45,7 @@ export class AuditLogsTableComponent implements OnInit, OnDestroy {
      * @memberof AuditLogsTableComponent
      */
     public ngOnInit(): void {
-        //
+        // this.getFilteredLogs();
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.ts:48]: please check  it

File: apps/web-giddh/src/app/create/components/invoice/templates/template.component.html [hunk 5]
```diff
@@ -20,7 +20,7 @@ <h1 class="text-center">Follow three easy steps below to create Invoice</h1>
     <!-- <section class="container">
 <div class="row">
   <div class="col-xs-12 alC">
-    <div class="pr mrB4 clearfix mrT2"> -->
+    <div class="pr mrB4 clearfix mr-t2"> -->
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/create/components/invoice/templates/template.component.html:23]: remove commented code if not required

File: apps/web-giddh/src/app/invoice/create/invoice.create.component.html [hunk 6]
```diff
@@ -58,7 +58,7 @@
                         <textarea name="billingDetails.address" class="form-control" [(ngModel)]="invFormData.accountDetails.billingDetails.address"></textarea>
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/create/invoice.create.component.html:58]: remove commented  code if not required

File: apps/web-giddh/src/app/invoice/eWayBill/download-or-preview-eway/download-or-preview-eway.component.html [hunk 7]
```diff
@@ -2,7 +2,7 @@
     <!-- <div *ngIf="isErrOccured$ | async">
         <p class="lead alC">Oops! Something went wrong.</p>
     </div> -->
-    <!-- <div class="loader mrT4" *ngIf="!showPdfWrap">
+    <!-- <div class="loader mr-t4" *ngIf="!showPdfWrap">
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/eWayBill/download-or-preview-eway/download-or-preview-eway.component.html:5]: remove commented code if not required, why we are updating commented code

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 8]
```diff
@@ -598,11 +604,11 @@ ul.sub-list {
     max-height: 290px;
 }
 
-.upgradePlan.list-inline > li {
+.upgrade-plan.list-inline > li {
     padding-left: 0 !important;
 }
 
-.nav-left-bar .company-name .cText span {
+.nav-left-bar .company-name .c-text span {
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.scss:611]: if we update the naming convention of class so we need to update class as a generic name c-text is not a generic name, update class name

File: apps/web-giddh/src/assets/css/style-1.scss [hunk 9]
```diff
@@ -736,15 +736,15 @@ aside-menu-account .container-fluid {
     text-transform: uppercase;
 }
 
-.txtLwc {
+.txt-lwc {
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/assets/css/style-1.scss:739]: if we update the naming convention of class so we need to update class as a generic name txt-lwcs not a generic name, update class name

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 10]
```diff
@@ -1829,15 +1866,15 @@ input:-moz-placeholder {
...
+.company-name ul li > span {
     color: #333 !important;
     text-overflow: ellipsis;
     white-space: nowrap;
     overflow: hidden;
     font-size: 14px;
 }
 
-.company-name .cText .cName {
+.company-name .c-text .c-name {
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/assets/css/style-2.scss:1877]: Add generic class name

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html [hunk 11]
```diff
@@ -455,12 +454,15 @@
 
         <div class="row">
 
-            <div class="col-xs-12 clearfix mrT1">
+            <div class="col-xs-12 col-md-12 clearfix mr-t1">
 
                 <div class="mrR1 form-group pull-left cp">
 
                     <label class="pull-left cp fs16">
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html]: Resolved

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 10 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/assets/css/style-2.scss:1877]: generic class name will be covered in code designing

File: apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss [hunk 12]
```diff
@@ -0,0 +1,20 @@
+.list-item:focus,
+.list-item:focus .list-item {
+    color: #fff !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss:3]: use color variable

File: apps/web-giddh/src/app/theme/ng2-vs-for/ng2-vs-for.ts [hunk 13]
```diff
@@ -209,10 +209,12 @@ export class VsForDirective implements OnChanges, AfterViewInit, OnDestroy {
     }
 
     public ngAfterViewInit() {
-        // this.vsScrollParent, this.vsAutoresize, this.tagName, this.__horizontal);
         this.view = this._viewContainer.createEmbeddedView(this._templateRef);
+        console.log(this._element.nativeElement);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/ng2-vs-for/ng2-vs-for.ts:213]: remove console

**Review by ravinderwalkover — COMMENTED** [review]
Reviews

File: apps/web-giddh/src/app/invoice/eWayBill/download-or-preview-eway/download-or-preview-eway.component.html (hunk 7 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/eWayBill/download-or-preview-eway/download-or-preview-eway.component.html:5]: @rohit-walkover  this PR also contains code-designing part, but that was not completed, all files who has class = mrT4 is updated in this PR. so this commented code also got updated.

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by shalinee-walkover — APPROVED** [review]
Approved

File: apps/web-giddh/src/app/sales/sales.module.ts [hunk 14]
```diff
@@ -1,36 +1,38 @@
...
 import { AsideMenuSalesOtherTaxes } from './aside-menu-sales-other-taxes/aside-menu-sales-other-taxes';
+import { SalesInvoiceComponent } from './create/sales.invoice.component';
+import { DiscountListComponent } from './discount-list/discountList.component';
+import { SalesAsideMenuAccountComponent } from './sales-aside-menu-account/sales.aside.menu.account.component';
+import { SalesComponent } from './sales.component';
+import { SalesRoutingModule } from './sales.routing.module';
 
+// import { TooltipModule, TypeaheadModule, CollapseModule } from 'ngx-bootstrap';
+// import { PaginationModule  } from 'ngx-bootstrap/pagination';
+// import { TabsModule } from 'ngx-bootstrap/tabs';
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/sales/sales.module.ts:35]: remove commented code

File: apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss [hunk 15]
```diff
@@ -0,0 +1,21 @@
+@import "../../../../../assets/css/color.scss";
+.list-item:focus,
+.list-item:focus .list-item {
+    color: $white !important;
+    background: #2d9ee0;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss:5]: Do use color var

File: apps/web-giddh/src/app/search/search.component.html [hunk 16]
```diff
@@ -1,8 +1,8 @@
 <div class="container-fluid">
-    <div class="">
+    <div>
         <div class="clearfix ">
             <div class="top-bar clearfix bdrB">
-                <h1 class="section-title inline"><strong>Search</strong></h1>
+                <!-- <h1 class="section-title inline"><strong>Search</strong></h1> -->
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/search/search.component.html:5]: please check once we have commented **search** label

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 17]
```diff
@@ -717,6 +724,42 @@ ul.sub-list {
...
+}
+.create-company-btn{
+    border-top: 1px solid $gainsboro;
+}
+.user-dropdown-header:after {
+    position: absolute;
+    content: '';
+    height: 20px;
+    width: 20px;
+    background-color: #e4e4e4;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.scss:756]: Do use color var

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by rohit-walkover — APPROVED** [review]

**Review by rohit-walkover — CHANGES_REQUESTED** [review]
reviewed Please make changes angular.json file web-giddh prod architecture, webpack files and local env prod  files according to sub domain information(if changing)

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 18]
```diff
@@ -337,7 +336,7 @@ button.btn.btn-filter {
     background: $whisper-color;
     border: 1px solid $whisper-color;
     font-size: 14px;
-    color: $midnight-blue;
+    color: #1a237e;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/contact/contact.component.scss]: done

File: apps/web-giddh/src/app/inventory-in-out/components/inventory-in-out-report/inventory-in-out-report.component.html [hunk 19]
```diff
@@ -1,5 +1,5 @@
 <section class="h100">
-    <form class="fb__1-container mrB1 sadik" name="groupReportForm" novalidate (ngSubmit)="applyFilters(1)" #f="ngForm">
+    <form class="fb__1-container mr-b1 sadik" name="groupReportForm" novalidate (ngSubmit)="applyFilters(1)" #f="ngForm">
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/inventory-in-out/components/inventory-in-out-report/inventory-in-out-report.component.html]: done

File: apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss (hunk 12 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss]: done

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 17 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/shared/header/header.component.scss]: done

File: apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss (hunk 15 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss]: done

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.scss [hunk 20]
```diff
@@ -16,6 +16,28 @@
...
+        .d-inline-flex {
+            display: inline-flex;
+        }
+
+        .custom-estimation label {
+            position: absolute;
+            top: -24px;
+            right: 15px;
+            font-size: 12px;
+            color: #666666;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.scss]: done

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.html [hunk 21]
```diff
@@ -1,29 +1,29 @@
...
-                    <div class="advance-icon" tooltip="Advance Search">
-                        <span class="icon-advance-filter" (click)="toggleAdvanceSearchPopup()"></span>
+                    <div class="advance-icon"  (click)="toggleAdvanceSearchPopup()" tooltip="Advance Search">
+                        <span class="icon-advance-filter"></span>
                     </div>
 
                     <span style="line-height: 2.1" class="ml-1" *ngIf="isAdvanceSearchApplied">
                         <a class="cp" href="javascript: void 0" (click)="resetAdvanceSearch()">
-                            <i aria-hidden="true" class="glyphicon glyphicon-refresh"
+                            <i aria-hidden="true" class="icon-refrsh"
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/contact/aging-report/aging-report.component.html]: done

File: apps/web-giddh/src/assets/css/style-1.scss (hunk 9 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/assets/css/style-1.scss:739]: generic class name will be covered in code designing

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 8 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/shared/header/header.component.scss:611]: generic class name will be covered in code designing

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-form/audit-logs-form.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/audit-logs/components/audit-logs-form/audit-logs-form.component.html:37]: Dont'remove

**Review by rohit-walkover — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**